packages/hoppscotch-cli/src/utils/mutators.ts

```diff
@@ -39,49 +34,46 @@ export const parseErrorMessage = (e: unknown) => {
   return msg.replace(/\n+$|\s{2,}/g, "").trim();
 };
 
+export async function readJsonFile(path: string) {
```

Comment:
Set the return type to `unknown` to tell TypeScript that we don't know the exact type of the value that will be returned (forces us to do validation).

```suggestion
export async function readJsonFile(path: string): unknown {
```

Reply:
Unknown return type at https://github.com/hoppscotch/hoppscotch/pull/2907/commits/e94fb04215071caf6ab5b1910d6df0a6a63294a1#diff-fccb5e9c3967b37fa6f0242a9bdddcfcad1f7e7baf5f4508f95730ccf78bdf97R37